README.md

```diff
@@ -4,22 +4,22 @@
 [![.github/workflows/packages.yml](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/packages.yml/badge.svg)](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/packages.yml)
 [![Mergin Plugin Tests](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/run-test.yml/badge.svg)](https://github.com/lutraconsulting/qgis-mergin-plugin/actions/workflows/run-test.yml)
 
-QGIS plugin aims to simplify handling of [Mergin Maps](https://public.cloudmergin.com/) projects.
+QGIS plugin aims to simplify handling of [Mergin Maps](https://app.merginmaps.com/) projects.
 
-[Mergin Maps](https://public.cloudmergin.com/) is a service to store, synchronise and manage GIS data and QGIS projects. Currently, [Mergin Maps](https://public.cloudmergin.com/) is used as the back-end of [Mergin Maps Input](https://inputapp.io/), an [open source](https://github.com/lutraconsulting/input) mobile application to collect data in field.
+[Mergin Maps](https://app.merginmaps.com/) is a service to store, synchronise and manage GIS data and QGIS projects. Currently, [Mergin Maps](https://app.merginmaps.com/) is used as the back-end of [Mergin Maps Input](https://inputapp.io/), an [open source](https://github.com/lutraconsulting/input) mobile application to collect data in field.
```

Comment:
best to reword this -> there is not Input as such anymore and inputapp.io address is now only redirected